net/roamguide/files/etc/config/roamguide

```diff
@@ -0,0 +1,11 @@
+# repeat the following for every wireless AP device you like to guide.
```

Comment:
i would prefer a solution where this package detects itself if there is a single-band or a dual-band device, which in the latter case would need two sections for client0 and client1

Reply:
I opened an issue here: https://github.com/sargon/gluon-sargon/issues/14

Reply:
This issue is said to be fixed in https://github.com/sargon/gluon-sargon/commit/c9539a04b171d246d9643608c59f1a91ddb2d911 (though it just creates both and fails silently on single band devices).